pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -418,7 +418,7 @@ stdenv.mkDerivation {
     "-Dnetworkd=${lib.boolToString withNetworkd}"
     "-Doomd=${lib.boolToString withOomd}"
     "-Dpolkit=${lib.boolToString withPolkit}"
-    "-Dcryptsetup=${lib.boolToString withCryptsetup}"
+    "-Dlibcryptsetup=${lib.boolToString withCryptsetup}"
```

Comment:
Example case where we had an incorrect/outdated name of a meson option.

Reply:
Doesn't look like systemd ever had a "cryptsetup" option, only "libcryptsetup".